Sources/StreamChatUI/ChatMessageList/Attachments/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -80,72 +75,64 @@ extension _ChatMessageFileAttachmentListView {
 
         override open func setUpLayout() {
             super.setUpLayout()
-
-            addSubview(fileIconImageView)
-            addSubview(actionIconImageView)
-            addSubview(fileNameAndSizeStack)
-
-            NSLayoutConstraint.activate([
-                fileIconImageView.leadingAnchor.pin(equalTo: layoutMarginsGuide.leadingAnchor),
-                fileIconImageView.topAnchor.pin(equalTo: layoutMarginsGuide.topAnchor),
-                fileIconImageView.bottomAnchor.pin(equalTo: layoutMarginsGuide.bottomAnchor),
-                
-                actionIconImageView.topAnchor.pin(equalTo: layoutMarginsGuide.topAnchor),
-                actionIconImageView.trailingAnchor.pin(equalTo: layoutMarginsGuide.trailingAnchor),
-                actionIconImageView.leadingAnchor.pin(
-                    equalToSystemSpacingAfter: fileNameAndSizeStack.trailingAnchor,
-                    multiplier: 1
-                ),
-                
-                fileNameAndSizeStack.leadingAnchor.pin(
-                    equalToSystemSpacingAfter: fileIconImageView.trailingAnchor,
-                    multiplier: 2
-                ),
-                fileNameAndSizeStack.centerYAnchor.pin(equalTo: centerYAnchor),
-                fileNameAndSizeStack.topAnchor.pin(greaterThanOrEqualTo: layoutMarginsGuide.topAnchor),
-                fileNameAndSizeStack.bottomAnchor.pin(lessThanOrEqualTo: layoutMarginsGuide.bottomAnchor)
-            ])
+            addSubview(mainContainerStackView)
+            mainContainerStackView.pin(to: layoutMarginsGuide)
+            
+            spinnerAndSizeStack.addArrangedSubviews([loadingIndicator, fileSizeLabel])
+            fileNameAndSizeStack.addArrangedSubviews([fileNameLabel, spinnerAndSizeStack])
+            mainContainerStackView.addArrangedSubviews([fileIconImageView, fileNameAndSizeStack, actionIconImageView])
+            
+            spinnerAndSizeStack.axis = .horizontal
+            spinnerAndSizeStack.spacing = 8
```

Comment:
Please use the default spacing instead

Reply:
Hmm I probably do not need to set the spacing, because the default spacing `.auto` is default value. Deleted the both lines for main and spinnerAndSize stacks...